src/js_native_api.h

```diff
@@ -118,6 +118,16 @@ NAPI_EXTERN napi_status NAPI_CDECL node_api_create_property_key_utf16(
     napi_env env, const char16_t* str, size_t length, napi_value* result);
 #endif  // NAPI_EXPERIMENTAL
 
+#ifdef NAPI_EXPERIMENTAL
+#define NODE_API_EXPERIMENTAL_HAS_PROPERTY_KEYS
```

Comment:
```suggestion
#define NODE_API_EXPERIMENTAL_HAS_SET_NAMED_PROPERTY_LEN
```

This value is less generic and thus more tied to the specific API we're adding.